lib/smart_answer_flows/marriage-abroad/outcomes/_synonyms_of_cp.govspeak.erb

```diff
@@ -114,7 +114,7 @@
   - ’domestic partnership’ (in Nova Scotia)
   - ’union civile’, or ’civil union’ (in Quebec)
 <% when 'new-zealand' %>
-  You may be able to get married or enter into a ‘civil union’ at the British embassy or consulate in New Zealand. Civil unions are recognised as civil partnerships under British law.
+  Civil unions are recognised as civil partnerships under British law.
```

Comment:
Ok cool, I'll start doing that then. In which case I guess I'll link to the relevant ZenDesk ticket - so for this change it's https://govuk.zendesk.com/agent/tickets/1300890